src/Tizen.Sensor/Tizen.Sensor/Sensor.cs

```diff
@@ -541,9 +689,778 @@ private bool CheckSensorHandle()
+        {
+            IntPtr list;
+            IntPtr[] sensorList;
+            int count;
+            int error = Interop.SensorManager.GetSensorListByUri(uri, out list, out count);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error getting sensor list by uri");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.GetSensorListByUri Failed");
+            }
+            sensorList = Interop.IntPtrToIntPtrArray(list, count);
+            _sensorHandle = sensorList[index];
+            Interop.Libc.Free(list);
+        }
+
+        /// <summary>
+        /// Creates a recorder option handle.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        private void RecorderCreateOption()
+        {
+            int error = Interop.SensorRecoder.RecorderCreateOption(out recorderOption);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error creating the sensor recorder option");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderCreateOption Failed");
+            }
+        }
+
+        /// <summary>
+        /// Creates a recorder query handle.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        private void RecorderCreateQuery()
+        {
+            int error = Interop.SensorRecoder.RecorderCreateQuery(out recorderQuery);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error creating the sensor recorder query");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderCreateQuery Failed");
+            }
+        }
+
+        /// <summary>
+        /// Gets a double value from a record data.
+        /// </summary>
+        /// <param name="data">
+        /// Record data handle.
+        /// </param>
+        /// <param name="key">
+        /// Data attribute to retrieve.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public double RecorderDataGetDouble(int data, int key)
+        {
+            double value;
+            int error = Interop.SensorRecoder.RecorderDataGetDouble(data, key, out value);
+            if (error != (int)SensorError.None)
+            {
+                Log.Error(Globals.LogTag, "Error getting the sensor recorder double data");
+                throw SensorErrorFactory.CheckAndThrowException(error, "Sensor.RecorderDataGetDouble Failed");
+            }
+            return value;
+        }
+
+        /// <summary>
+        /// Gets the start and the end time of the time period of a given record data.
+        /// </summary>
+        /// <param name="data">
+        /// Record data handle.
+        /// </param>
+        /// <since_tizen> 6 </since_tizen>
+        public Tuple<long, long> RecorderDataGetTime(int data)
```

Comment:
There is no description about return value

Reply:
description about return value added.